lib/radioactive_bunnies/worker.rb

```diff
@@ -1,17 +1,14 @@
 require 'atomic'
 require 'radioactive_bunnies/context'
-
+require 'radioactive_bunnies/deadletter_worker'
 module RadioactiveBunnies::Worker
 
-  def ack!
```

Comment:
Based on convo, I vote we don't delete it here, since its unrelated to the DLX task. Lets review how we would move service and RpcWorker implementers to use an ack(payload), and tackle as a separate PR.